modules/text2colbert-jinaai/ent/class_settings.go

```diff
@@ -0,0 +1,115 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package ent
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/weaviate/weaviate/entities/models"
+	"github.com/weaviate/weaviate/entities/moduletools"
+	"github.com/weaviate/weaviate/entities/schema"
+	basesettings "github.com/weaviate/weaviate/usecases/modulecomponents/settings"
+)
+
+const (
+	// Default values for URL (model is ok) cannot be changed before we solve how old classes that have the defaults
+	// NOT set will handle the change
+	DefaultJinaAIModel           = "jina-colbert-v2"
+	DefaultVectorizeClassName    = true
+	DefaultPropertyIndexed       = true
+	DefaultVectorizePropertyName = false
+	DefaultBaseURL               = "https://api.jina.ai"
+	LowerCaseInput               = false
+)
+
+var DefaultDimensions int64 = 128
```

Comment:
Can you add this to the class config defaults here: modules/text2colbert-jinaai/config.go?

Reply:
good catch! done